app/main.py

```diff
@@ -1,2 +1,87 @@
+from dataclasses import dataclass
+from typing import Hashable, Any, Optional, Iterator
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` should be a float representing the load factor, not an integer division. Consider changing it to `THRESHOLD = 2 / 3.0` or `THRESHOLD = 0.67`.